packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -142,15 +145,21 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
     <SquadPostPageNavigation squadLink={post.source.permalink} />
   );
   const navigation: Record<PostType, ReactNode> = {
-    article: null,
+    article: !!router?.query?.squad && (
+      <Link href={`/squads/${router.query.squad}`}>
```

Comment:
@capJavert this is not correct now, you made it the name, but squad handle could be different.
That's why i added both, one for the url and one for text in the link.
Or did i miss something in squad urls maybe?

Reply:
Should be fixed now, completely missed the part of logic.